libdrgn/dwarf_index.c

```diff
@@ -1834,10 +1872,10 @@ index_specification(struct drgn_dwarf_index *dindex, uintptr_t declaration,
  * DW_AT_specification.
  */
 static struct drgn_error *index_cu_first_pass(struct drgn_dwarf_index *dindex,
-					      struct drgn_dwarf_index_cu *cu)
+					      struct drgn_dwarf_index_cu *cu,
+					      const char* ptr)
```

Comment:
Since you changed it so that `index_cu_first_pass()` is only done for the initial indexing, you don't need the `ptr` parameter anymore.